crates/ibc/src/core/ics04_channel/msgs/acknowledgement.rs

```diff
@@ -47,6 +47,12 @@ impl AsRef<[u8]> for Acknowledgement {
     }
 }
 
+impl From<&[u8]> for Acknowledgement {
+    fn from(bytes: &[u8]) -> Self {
+        Self(bytes.to_vec())
+    }
+}
+
```

Comment:
Is this impl `From<&[u8]>` needed? It is used for tests which also can be fulfilled by `From<Vec<u8>>` derived [here](https://github.com/cosmos/ibc-rs/blob/c5415e637fb112382d25d000075de8a7ec300a34/crates/ibc/src/core/ics04_channel/msgs/acknowledgement.rs#L30)

Reply:
not needed indeed [887c4a2](https://github.com/cosmos/ibc-rs/pull/364/commits/887c4a2846bb9c18ec102c5f472bdb37e46641a1)